test/shapes/test_circle.py

```diff
@@ -0,0 +1,74 @@
+from pathlib import Path
+
+import fpdf
+from test.conftest import assert_pdf_equal
+
+
+HERE = Path(__file__).resolve().parent
+
+
+def next_row(pdf):
+    pdf.ln()
+    pdf.set_y(pdf.get_y() + size + margin)
+
+
+size = 50
+margin = 10
+
+
+def test_circle_style(tmp_path):
+    pdf = fpdf.FPDF(unit="mm")
+    pdf.add_page()
+
+    for counter, style in enumerate(["", "F", "FD", "DF", None]):
+        pdf.circle(x=pdf.get_x(), y=pdf.get_y(), r=size, style=style)
+        pdf.set_x(pdf.get_x() + size + margin)
+        if counter % 3 == 0:
```

Comment:
Note that with `counter % 3 == 2` it would be slightly nicer, as you'd get a first row with 3 circles instead of only one.